### PR TITLE
Changed vagrant provisioning to ansible

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

**Comment by chives** [comment]
@fsi-open

File: vagrant/provisioning/roles/symfony2/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- name: Create prod database
+  mysql_db:
+    name: "{{ item }}"
+    state: present
+    collation: utf8_general_ci
+  with_items: project_databases
+
+- name: create static resource dir
+  file: path=/var/www/content-block-bundle/features/fixtures/project/web/bundles
+        state=directory
```
Comment (szymach) [inline vagrant/provisioning/roles/symfony2/tasks/main.yml]: isn't this indentation wrong?

Comment (chives) [inline vagrant/provisioning/roles/symfony2/tasks/main.yml]: nope, it's a single line

**Comment by chives** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1:

**Comment by rn0** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Pull request merged**